src/MSBuild/TerminalLogger/TerminalLogger.cs

```diff
@@ -709,8 +855,6 @@ internal void DisplayNodes()
         Terminal.Write(AnsiCodes.HideCursor);
         try
         {
-            // Move cursor back to 1st line of nodes.
-            Terminal.WriteLine($"{AnsiCodes.CSI}{_currentFrame.NodesCount + 1}{AnsiCodes.MoveUpToLineStart}");
```

Comment:
Moved into rendering the node update string so we can see it in unit tests. Kept hide cursor and show cursor here, for the same reason.